extensions/spark/kyuubi-spark-authz/src/main/scala/org/apache/kyuubi/plugin/spark/authz/ranger/SparkRangerAdminPlugin.scala

```diff
@@ -78,6 +78,10 @@ object SparkRangerAdminPlugin extends RangerBasePlugin("spark", "sparkSql")
         if (plugin != null) {
           LOG.info(s"clean up ranger plugin, appId: ${plugin.getAppId}")
           plugin.cleanup()
+          val auditProviderFactory = plugin.getAuditProviderFactory
+          if (auditProviderFactory != null) {
```

Comment:
LGTM overall excpet a minor comment.

AFAIK, `auditProviderFactory ` should not be null, maybe we can simplify it to `plugin.getAuditProviderFactory.shutdown()`.

Reply:
Done @Yikf